content/blog/html/2018/12/23/deploying_velociraptor_with_oauth_sso/_index.md

```diff
@@ -202,7 +202,7 @@ candy :-).
 
 {{% notice note Thanks %}}
 
-Shouts to the folks from [Klein & Co](https://www.kleinco.com.au/) who
```

Comment:
This is an old post which was accurate at the time. We probably dont need to rewrite history :-)